csharp/src/Ice/Communicator.cs

```diff
@@ -943,12 +952,12 @@ public Reference CreateReference(string s, string? propertyPrefix = null)
 
                     case 's':
                         {
+                            // we ignore this secure option, just parse it
```

Comment:
Shouldn't we print a warning to explain that it's no longer supported?

Reply:
Added warning.